Refactor identified enums in Reports (#11122)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/directoryaudit.md

| initiatedBy         | [auditActivityInitiator](auditactivityinitiator.md) | Indicates information about the user or app initiated the activity.                                                                                                                                                                                                    |
| loggedByService     | String                                              | Indicates information on which service initiated the activity (For example: Self-service Password Management, Core Directory, B2C, Invited Users, Microsoft Identity Manager, Privileged Identity Management.                                                          |
| result              | operationResult                                              | Indicates the result of the activity. Possible values are: `success`, `failure`, `timeout`, `unknownFutureValue`.                                                                                                                                                       |

> | resultReason        | String                                              | Indicates the reason for failure if the result is "failure" or "timeout".                                                                                                                                                                                              |

Reviewer: ```suggestion
| resultReason        | String                                              | Indicates the reason for failure. Possible values: `failure` or `timeout`. |
```
